lib/ansible/modules/web_infrastructure/jira.py

```diff
@@ -390,6 +397,7 @@ def main():
             linktype=dict(),
             inwardissue=dict(),
             outwardissue=dict(),
+            timeout=dict(type='float', default=10),
```

Comment:
As i can see, `timeout` param passes to python `socket` module, which uses float for timeout: https://docs.python.org/3/library/socket.html#socket.socket.settimeout
Please, tell me if i'm wrong.